### DISCUSSION TITLE
Getting "Vercel Security Checkpoint" Page on every load

### DISCUSSION THREAD

**Discussion by thhmoc67** [vercel/next.js#59436] [Help] [answered]
### Summary

<img width="1512" alt="Screenshot 2023-12-09 at 3 06 14 PM" src="https://github.com/vercel/next.js/assets/15047392/32db4d1f-fa24-4aa0-a8c4-8b8804b70318">

**I am getting this screen only on my production website (not on staging or other domain deployments). 

once I load the page and reload, then again this page shows for a few minutes.**

### Example

https://thehelloworld.com

**Comment by caseyocampo** [comment]
You may have [Attack Challenge Mode](https://vercel.com/docs/security/attack-challenge-mode) enabled. Try disabling.

For anyone else who has this issue even if this is disabled, I'm having that same issue. Anyone know if this is a bug? Or how to fix it?

**Reply by talglobus** [reply]
Did you figure out what this was?

**Reply by caseyocampo** [reply]
Hey Tal, I did have Attack Challenge Mode on and then disabled it. I think I had to purge my cache and wait for it to clear to get it to stop showing. It's been a long time though and I no longer and I'm not sure if that was the fix.

**Comment by Nuraumar17** [comment]
Param

**Comment by lufog** [comment]
On my Linux machine, I always get:
```
Vercel Security Checkpoint
Failed to verify your device. Please refresh and try again.
```

**Comment by talglobus** [comment]
Did anyone else find a solution to this? Currently running into this issue on a live production application

**Comment by brandonvilla21** [comment]
I am getting that issue and browsers are not being verified. This is prod application

**Comment by nathanielmhld** [comment]
Yeah, huge issue for us. All text streaming is now broken.

**Reply by nathanielmhld** [reply]
Seems to be getting fixed

**Comment by ntotten** [comment]
Happening to us also, many customer sites are not loading.

**Comment by Lakshay122** [comment]
Hi Guys, is there anyone to sovle this ?

**Comment by bjones256** [comment]
getting this today.

**Reply by Girbi** [reply]
Same, i thought it was from me.
Edit: now it works, maybe there was a small downtime.

**Reply by bjones256** [reply]
I closed and reopened chrome and it worked

**Comment by yihuanlin** [comment]
I have [Attack Challenge Mode](https://vercel.com/docs/security/attack-challenge-mode) disabled, enabled then disabled again. Did not help. I keep getting "Failed to verify your browser" and my IP is marked as DDoS attack.

**Reply by yihuanlin** [reply]
> I have [Attack Challenge Mode](https://vercel.com/docs/security/attack-challenge-mode) disabled, enabled then disabled again. Did not help. I keep getting "Failed to verify your browser" and my IP is marked as DDoS attack.

I am using additional CDN and I guess that's the issue. I ignored caching on ./well-known and am no longer getting the error.

**Comment by maxpc55** [comment, marked as answer]
for those who are experiencing this, try closing and reopening the page or browser, or try manually clearing the cookies... if it still doesn't work, disable all the extensions you have installed, this may help, if you have several extensions one or more of them may generate a conflict in this verification system... you can disable them one by one to find out which one may be causing this... I hope I helped

**Comment by t77bsh** [comment]
this happening to me even though I don't have Attack Challenger Mode enabled

**Reply by yihuanlin** [reply]
If your site is using cdn you might want to ignore caching on ./well-known

**Reply by t77bsh** [reply]
I'm just using vercel no other CDN

**Comment by ChristinWhite** [comment]
Running into this issue on Vivaldi and Floorp, I had to switch to Comet just to read some articles.

This is very annoying, and I'll have to think long and hard about ever depending on Vercel as part of my stack.

It's also rather Ironic given that Comet is specifically designed for agentic browsing and, of the three browsers I tried, by for the most likely to not actually have a human driving.

**Reply by adrienthebo** [reply]
> Running into this issue on Vivaldi and Floorp

I just tested both of these and they work fine.

If anyone is experiencing these issue, please report your browser version and a list of installed addons and noteworthy customizations.

**Comment by norandom** [comment]
Issue is back. Will stop using Vercel everywhere. Customers complain now.
